src/pages/modules/[...moduleslug].astro

```diff
@@ -24,6 +24,12 @@ const { moduleslug } = Astro.params;
 const { module } = Astro.props;
 const tool = module.meta.tools[0];
 const toolName = Object.keys(tool)[0];
+const response = await fetch(
+    'https://raw.githubusercontent.com/nf-core/modules/master/tests/modules/nf-core/' +
+        module.name.replace('_', '/') +
+        '/main.nf'
+);
+const code = await response.text();
```

Comment:
could you use octokit to reduce the danger of hitting the API limit for this? Also probably best if we do this as a prebuild step in `components.json.ts`.